src/universal/modules/landing/ducks/auth.js

```diff
@@ -0,0 +1,27 @@
+const SET_AUTH_TOKEN = 'notifications/SET_AUTH_TOKEN';
```

Comment:
I think giving it's own reducer makes a TON of sense.

Do you think it should be nested under the landing module or up at the `universal/` level?